src/common/minting/commands/common/mint-nft-command.spec.ts

```diff
@@ -0,0 +1,40 @@
+// import MintError from '../../enums/mint-error';
```

Comment:
Please clean this up as well, make sure to not include commented code and or console log statements in the code